here_for_change/municipalities/views.py

```diff
@@ -287,3 +289,15 @@ def get(self,request):
         else:
             return redirect("home")
 
+
+class Feedback(View):
```

Comment:
@cyber237 please save this form data to the db, associate with ward

Reply:
I am on it

Reply:
@paulmwatson This is already done, check main.

<img width="1680" alt="Screenshot 2023-01-11 at 23 34 30" src="https://user-images.githubusercontent.com/58531874/211933046-60514824-b5f3-4b19-b244-0082be740797.png">

<img width="1680" alt="Screenshot 2023-01-11 at 23 34 07" src="https://user-images.githubusercontent.com/58531874/211933034-ba8645f7-2d94-47e5-8c78-f47e4a305df4.png">